packages/node-core/src/configure/migration-service/migration.ts

```diff
@@ -18,60 +61,115 @@ import {
 } from '../../utils/sync-helper';
 import {NodeConfig} from '../NodeConfig';
 
+type RemovedIndexes = Record<string, IndexesOptions[]>;
+
+const logger = getLogger('db-manager');
+
 export class Migration {
   private sequelizeModels: ModelStatic<any>[] = [];
-  private rawQueries: string[] = [];
+  private mainQueries: string[] = [];
   private readonly historical: boolean;
+  private extraQueries: string[] = [];
```

Comment:
Comments explaining the reason for 2 sets would be good
```suggestion
  private mainQueries: string[] = [];
  private extraQueries: string[] = [];
  private readonly historical: boolean;

```